tests/integration/targets/dcnm_interface/tests/dcnm/dcnm_eth_delete.yaml

```diff
@@ -214,7 +214,7 @@
           - 'result.changed == false'
           - '(result["diff"][0]["merged"] | length) == 0'
           - '(result["diff"][0]["deleted"] | length) == 0'
-          - '(result["diff"][0]["replaced"] | length) == 0'
+          - '(result["diff"][0]["replaced"] | length) == 5'
```

Comment:
Maybe I missed this but can you add a testcase to `delete/default` a single ethernet interface and also one to `delete/default` two ethernet interfaces?  Want to make sure we have test coverage to 

* Default all ethernet interfaces
* Delete one ethernet interfaces
* Delete multiple ethernet interfaces